packages/js-moi-signer/dist/signer.js

```diff
@@ -104,6 +104,8 @@ class Signer {
      * performing validity checks.
      *
      * @param {InteractionObject} ixObject - The interaction object to prepare.
+     * @param {Options} options - The options for preparing the interaction object. (optional)
+     * @param {object} options.ignore - The fields to ignore validation during preparation. (optional)
```

Comment:
why is this required?

Reply:
removed.

commit b2eb84c76d35164ed14f16d0312a9c3a86ead0dd